src/Microsoft.Extensions.Logging.Console/ConsoleLogScope.cs

```diff
@@ -6,6 +6,7 @@
 
 namespace Microsoft.Extensions.Logging.Console
 {
+    [Obsolete("Use " + nameof(LoggerExternalScopeProvider) + "instead" )]
```

Comment:
Why was this ever public?

Reply:
![https://media.giphy.com/media/fKk2I5iiWGN0I/giphy.gif](https://media.giphy.com/media/fKk2I5iiWGN0I/giphy.gif)

Reply:
Ok then :)